python/ray/services.py

```diff
@@ -789,3 +814,21 @@ def start_ray_head(address_info=None,
                              start_workers_from_local_scheduler=start_workers_from_local_scheduler,
                              num_cpus=num_cpus,
                              num_gpus=num_gpus)
```

Comment:
`redirect_stdout` and `redirect_stderr` take paths as arguments. Functions such as `start_ray_processes`, `start_ray_head`, and `start_ray_node` need multiple output paths for multiple processes, so we are taking the `redirect_output` flag and generating paths from there. Right now redirecting to `/dev/null` would require modifying these functions.

Perhaps a better way is just to stick with `redirect_output` and take the path-providing function (e.g., `new_log_files(name)` as an argument).